skills/lookup.js

```diff
@@ -0,0 +1,53 @@
+const ExpressionList = require('../submodules/Regex/ExpressionList');
+const ping = require('ping');
+
+
+module.exports = controller => {
+    controller.hears(
+        [ExpressionList.lookup], 
+        'direct_message', 
```

Comment:
Can you change this to: `'direct_message,direct_mention'` ? This means we can also just tag jarvis anywhere else, e.g mid thread `@jarvis lookup aps-tsiebler-vm`